app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -778,7 +778,7 @@ public void onResume() {
         }
 
         if (!initDone) {
-            if (updateLoyaltyCard) {
+            if (updateLoyaltyCard || duplicateFromLoyaltyCardId) {
                 setTitle(R.string.editCardTitle);
```

Comment:
Probably should move this title code out of the if update/duplicate part and just start (!initDone) with:

```
if (updateLoyaltyCard) {
    setTitle(R.string.editCardTitle);
} else {
   setTitle(R.string.addCardTitle);
}
```

Then the 

```
else {
    setTitle(R.string.addCardTitle);
}
```

a bit lower can be removed.

Reply:
But if i remove this bunch of code:

```[java]

        if (!initDone) {
            if (updateLoyaltyCard || duplicateFromLoyaltyCardId) {
                setTitle(R.string.editCardTitle);

                if (!mFrontImageUnsaved && !croppedFrontImage() && !mFrontImageRemoved) {
                    setCardImage(cardImageFront, Utils.retrieveCardImage(this, tempLoyaltyCard.id, ImageLocationType.front), true);
                }
                if (!mBackImageUnsaved && !croppedBackImage() && !mBackImageRemoved) {
                    setCardImage(cardImageBack, Utils.retrieveCardImage(this, tempLoyaltyCard.id, ImageLocationType.back), true);
                }
                if (!mIconUnsaved && !croppedIcon() && !mIconRemoved) {
                    setCardImage(thumbnail, Utils.retrieveCardImage(this, tempLoyaltyCard.id, ImageLocationType.icon), false);
                }
            } else {
                setTitle(R.string.addCardTitle);
            }

            if (mFrontImageUnsaved && !croppedFrontImage()) {
                setCardImage(cardImageFront, Utils.loadTempImage(this, TEMP_UNSAVED_FRONT_IMAGE_NAME), true);
            }
            if (mBackImageUnsaved && !croppedBackImage()) {
                setCardImage(cardImageBack, Utils.loadTempImage(this, TEMP_UNSAVED_BACK_IMAGE_NAME), true);
            }
            if (mIconUnsaved && !croppedIcon()) {
                setCardImage(thumbnail, Utils.loadTempImage(this, TEMP_UNSAVED_ICON_NAME), false);
            }
        }


```

the photos that are saved in each card won't be copied in the duplicated card